Index.html

```diff
@@ -38,7 +38,136 @@ <h2 class="greenheader">
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
+        </div>
+        <!--Second work item-->
+        <div class="work-item">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <div class="work-title">Multi-Post Stories Gain+Glory</div>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
+        </div>
+        <!--3rd work item-->
+        <div class="work-item">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <div class="work-title">Multi-Post Stories Gain+Glory</div>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
+        </div>
+        <!--4th work item-->
+        <div class="work-item">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <div class="work-title">Multi-Post Stories Gain+Glory</div>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
+        </div>
+        <!--5th work item-->
+        <div class="work-item">
+            <div class="work-image"></div>
+            <div class="work-content">
+                <div class="work-title">Multi-Post Stories Gain+Glory</div>
+                <ul class="cards">
+                    <li>Ruby on rails</li>
+                    <li>CSS</li>
+                    <li>JavScript</li>
+                    <li>HTML</li>
+                </ul>                
+                <div class="see-project">See project</div>
+            </div>
+        </div>
+    </section>
+    <!--Works section finishes-->
+    <!--About Me section starts-->
+    <section class="aboutme">
+        <div class="bg-1">
+            <h2 class="aboutme-heading">About me</h2>
+        </div>
+        <p>
+            I can help you build a product, feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="bg-2">
+            <p class="resume-button">Get my resume</p>
```

Comment:
Remember that this **button** is going to be pressed by the user at some point, please use the correct tag for this element (a `<button>` tag) to follow best practices! 🥳